From 03ae574b49d4f9a56102dd1a2a455fab46cb21da Mon Sep 17 00:00:00 2001
Message-ID: <03ae574b49d4f9a56102dd1a2a455fab46cb21da.1745272461.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1745272461.git.sharlatanus@gmail.com>
References: <cover.1745272461.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 21 Apr 2025 10:24:12 +0100
Subject: [PATCH 13/32] gnu: python-stcal: Update to 1.12.0.

* gnu/packages/astronomy.scm (python-stcal): Update to 1.12.0.
[arguments] <test-flags>: Skip one more test, sort alphabetically.

Change-Id: I3cfd4a2a523fc526a4b209cbf60461612f2f3bec
---
 gnu/packages/astronomy.scm | 13 +++++++------
 1 file changed, 7 insertions(+), 6 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index eb24e39667..4ea0882c96 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -5741,13 +5741,13 @@ (define-public python-stsci-imagestats
 (define-public python-stcal
   (package
     (name "python-stcal")
-    (version "1.11.1")
+    (version "1.12.0")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "stcal" version))
        (sha256
-        (base32 "0xafli4b3wyimpaxmvb0h5bha5g5kg76s4hykk63wkxrjrd2m2vf"))))
+        (base32 "1wb1rvr0p00jhhpn2szqi9dn8vzmb8v12yb96h8x24mj08p5clwk"))))
     (build-system pyproject-build-system)
     (arguments
      (list
@@ -5756,12 +5756,13 @@ (define-public python-stcal
               "-k" (string-join
                     ;; Tests requiring network access.
                     (list "not test_absolute_align"
-                          "test_relative_align[True]"
-                          "test_relative_align[False]"
-                          "test_get_catalog"
                           "test_create_catalog"
                           "test_create_catalog_graceful_failure"
-                          "test_parse_refcat")
+                          "test_get_catalog"
+                          "test_parse_refcat"
+                          "test_parse_sky_centroid"
+                          "test_relative_align[False]"
+                          "test_relative_align[True]")
                     " and not "))
       #:phases
       #~(modify-phases %standard-phases
-- 
2.49.0

